src/include/planner/join_order/cardinality_estimator.h

```diff
@@ -27,28 +27,31 @@ class CardinalityEstimator {
     void addNodeIDDomAndStats(const transaction::Transaction* transaction,
```

Comment:
The main change in the cardinality estimator is to make all the functions take in a `LogicalOperator` as input instead of a plan, this allows it to work during the operator pass (when we want to estimate on cardinalities for operators in a plan instead of plans that are still in construction)